src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2427,6 +2427,21 @@ limitations under the License.
             <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"/>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-thread-border-router</name>
+        <domain>HRAP</domain>
+        <typeName>Matter Thread Border Router</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0xFFF10011</deviceId><!-- TODO: ID-TBD -->
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters lockOthers="true">
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"/>
+            <include cluster="Thread Network Diagnostics" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Thread Border Router Mangement" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```

Comment:
There is no "Thread Border Router Mangement" cluster in the spec.

Reply:
This PR is based on [PR#8984](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8984) on the spec, which introduces the "Thread Border Router Mangement" cluster.

Reply:
The SPEC PR was merged now. Mark this PR as ready.